docs/source/demos/finnegans_wake_demo.rst

```diff
@@ -0,0 +1,55 @@
+Finnegan's Wake Demo
+====================
+
+.. figure:: https://cdn-images-1.medium.com/max/600/0*b42NPOnflrY07rEf.jpg
+    :width: 100%
+
+Overview
+--------
+
+.. important::
+
+    This demo requires connecting to a running network. By default the demo is hardcoded to connect to the local demo fleet.
+
+
+This demo is an example of a NuCypher decentralized network allowing Alice to share
+data with Bob using proxy re-encryption. This enables the private sharing of data between
+participants in public consensus networks, without revealing data keys to intermediary entities.
+
+
++------+-----------+----------------------------------------------------------------------------------------------+
+| Step | Character | Operation                                                                                    |
++======+===========+==============================================================================================+
+| 1    | Alice     | Alice sets a Policy on the NuCypher network (2/3) and grants access to Bob                   |
++------+-----------+----------------------------------------------------------------------------------------------+
+| 2    | Alice     | Label and Alice's key public key provided to Bob                                             |
++------+-----------+----------------------------------------------------------------------------------------------+
+| 3    | Bob       | Bob joins the policy with Label and Alice's public key                                       |
++------+-----------+----------------------------------------------------------------------------------------------+
+| 4    | Enrico    | DataSource created for the policy                                                            |
++------+-----------+----------------------------------------------------------------------------------------------+
+| 5    | Enrico    | Each plaintext message gets encapsulated through the DataSource to messageKit                |
++------+-----------+----------------------------------------------------------------------------------------------+
+| 6    | Bob       | Bob receives and reconstructs the DataSource from Policy public key and DataSource public key|
++------+-----------+----------------------------------------------------------------------------------------------+
+| 7    | Bob       | Bob retrieves the original message form DataSource and MessageKit                            |
++------+-----------+----------------------------------------------------------------------------------------------+
+
+
+Install Nucypher
+----------------
+
+    Acquire the nucypher application code and install the dependencies.
+    For a full installation guide see the [NuCypher Installation Guide](../guides/installation_guide).
```

Comment:
I don't get a hyperlink here for the Installation Guide - but it could just be my machine 😄